L/libcxxwrap_julia/build_tarballs.jl

```diff
@@ -2,10 +2,15 @@
 # `julia build_tarballs.jl --help` to see a usage message.
 using BinaryBuilder, Pkg
 
+# See https://github.com/JuliaLang/Pkg.jl/issues/2942
+# Once this Pkg issue is resolved, this must be removed
+uuid = Base.UUID("a83860b7-747b-57cf-bf1f-3e79990d037f")
+delete!(Pkg.Types.get_last_stdlibs(v"1.6.3"), uuid)
+
 name = "libcxxwrap_julia"
-version = v"0.9.0"
+version = v"0.9.1"
```

Comment:
The source is still at 0.9.0 though

Reply:
Ah, yes, good catch. Maybe a rebuild would suffice? I prefer keeping source and JLL version the same from now on.

Reply:
Rebuild with different deps won't work, though I am not sure that's the case here... can't you just add a 0.9.1 tag in your repo? It can point to the same commit...

Reply:
The deps didn't change, unless I'm missing something?

Reply:
OK I am dropping the version then... hopefully it'll work